posixlib/src/main/resources/scala-native/unistd.c

```diff
@@ -34,6 +34,24 @@
 #define _SC_TRACE_USER_EVENT_MAX 0
 #endif // __FreeBSD__
 
+// seems that pledge and unveil is OpenBSD,
+// some work to port it to Linux is onging but it far from the end,
+// and the best status is Justine's work to port it into cosmopolitan.
+#ifndef __OpenBSD_
+
+#include <errno.h>
+
+int pledge(const char *promises, const char *execpromises) {
```

Comment:
We should not try to redefine it here under the original name, instead let's define it as `scalanative_pledge` and redirect Scala to it by adding `@name("scalanative_pledge")` annotation. Same for `unveil` function